cmd/soroban-rpc/main.go

```diff
@@ -21,7 +21,7 @@ import (
 
 func main() {
 	var endpoint, horizonURL, stellarCoreURL, binaryPath, configPath, friendbotURL, networkPassphrase, dbPath, captivecoreStoragePath string
-	var captiveCoreHTTPPort, ledgerEntryStorageTimeoutMinutes, maxEventsLimit uint
+	var captiveCoreHTTPPort, ledgerEntryStorageTimeoutMinutes, maxEventsLimit, defaultEventsLimit uint
```

Comment:
We may want to declare a `LocalConfig` variable instead of declaring every variable independently and then  initializing a local config, but maybe that's out of scope for this PR.